css/build.css

```
@@ -78,45 +78,76 @@
   margin-top: 4px;
 }
 
-[data-notification-inbox-1-0-0-id] .notification .title {
+[data-fliplet-viewer-notification-inbox-1-0-0-id] .notification .tags {
+  margin-top: 4px;
+}
+
+[data-fliplet-viewer-notification-inbox-1-0-0-id] .notification .tags .tag {
+  font-size: 12px;
+  font-weight: 600;
+  border-radius: 4px;
+  padding: 4px 10px;
+  text-transform: uppercase;
+  color: #474974;
+  background-color: #eae9ec;
+}
+
+[data-fliplet-viewer-notification-inbox-1-0-0-id] .notification .tags .tag + tag {
```

How have you tested this?

```suggestion
[data-fliplet-viewer-notification-inbox-1-0-0-id] .notification .tags .tag + .tag {
```